api/src/main/kotlin/com/openlattice/search/Auth0UserSearchFields.kt

```diff
@@ -0,0 +1,21 @@
+package com.openlattice.search
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.util.Optional
+
+// https://auth0.com/docs/users/user-search/user-search-query-syntax
+data class Auth0UserSearchFields(
+    @JsonProperty val email: Optional<String> = Optional.empty(),
+    @JsonProperty val name: Optional<String> = Optional.empty()
```

Comment:
is there a reason you're using optionals here instead of just defaulting to null?

Reply:
Yea... this is just a symptom of me not being as familiar/comfortable with Kotlin, but dealing with the nulls got messy earlier on in the development cycle. I can try switching back if you'd like.